src/commands/pause.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProvider } from "../api-provider";
+import { getTodayDate } from "../utils/get-today-date";
+import { log } from "../utils/log";
+import { askConfirm } from "./prompts/confirm";
+
+export const pause = async (apiProvider: ApiProvider) => {
+    const date = getTodayDate();
+    const harvestApi = apiProvider.getHarvestApi();
+
+    const entries = await harvestApi.getTimeEntries(date);
+
+    const runningTimeEntry = entries.find(e => e.running);
+
+    if (!runningTimeEntry) {
+        log.info("You do not have a running timer");
+        return;
+    }
+
+    log.info("About to pause timer:");
+
+    const textLines = runningTimeEntry.text.match(/[^\r\n]+/g);
+    textLines.forEach(log.info);
+
+    const confirm = await askConfirm();
```

Comment:
I don't think we should ask for confirmation to pause a timer. You're not really going to do this by accident, it doesn't commit anything, and I may want to pause, then run away.